src/assets/styles/_mixins.scss

```diff
@@ -0,0 +1,13 @@
+// Mixins
+
+@mixin borderShadow() {
+  box-shadow: 0 1px 0 var(--calcite-app-border);
+}
+
+@mixin lineClamp($lines: 2) {
+  display: -webkit-box;
```

Comment:
[This doesn't seem like a standard value for display.](https://developer.mozilla.org/en-US/docs/Web/CSS/display) Is this a kind of hack? Might be working adding a comment.

Should we wrap this in an @supports?

Reply:
I think we discussed a while ago that using @supports is mostly about ie11 no?

Reply:
IE11 is a poster-child use case, but no @supports is browser agnostic. It's a progressive enhancement tool that let's you say (if this is supported in this environment, use it) and you can apply graceful defaults for when it's not.